File: examples/SDK/python/WorkingWithTables/describe_table.py
from __future__ import print_function  # Python 2/3 compatibility
import boto3, pprint

dynamodb = boto3.resource('dynamodb', region_name='us-west-2')

#table = dynamodb.Table("Music") # Substitute your table name for RetailDatabase

response = dynamodb.meta.client.describe_table(
    TableName='Music'
)

pprint.pprint(response)

File: examples/SDK/node.js/sdk_v3/control_plane/WorkingWithTables/describe_table.js
const { DynamoDBClient, DescribeTableCommand } = require("@aws-sdk/client-dynamodb");

const dynamodb = new DynamoDBClient({ region: "us-west-2" });

const describeTable = async () => {
  const command = new DescribeTableCommand({ TableName: "Music" }); // Substitute your table name for "Music"
  const response = await dynamodb.send(command);

  //console.log(JSON.stringify(response, null, 2));
  const streamEnabled = JSON.stringify(response.Table.StreamSpecification.StreamEnabled, null, 2)
  console.log(streamEnabled);
};

describeTable().catch((error) => console.error(JSON.stringify(error, null, 2)));
